tensorflow/lite/experimental/delegates/hexagon/hexagon_delegate_kernel.cc

```diff
@@ -376,7 +376,7 @@ void HexagonDelegateKernel::PrintPerformanceData(Profiler* profiler) {
     if (node_id != -1 && op_type_id >= 0) {
       profiler->AddEvent((op_type_id < 0 ? "" : op_name.data()),
                          Profiler::EventType::OPERATOR_INVOKE_EVENT, node_id, 0,
-                         counter);
+                         counter / 1000);
```

Comment:
The returned value is in cycles not latency. I had a todo to find a way to map between cycles and latency. Did you find somewhere that it is in microseconds ?

Reply:
Good question. Yes, as you said, I remember seeing "cycles" somewhere. However, according to public nnlib source code, [doc](https://source.codeaurora.org/quic/hexagon_nn/nnlib/tree/docs/intro.txt#n305) of `hexagon_nn_reset_perfinfo()`, and measured numbers on my Pixel phones, it seems to me it's in microseconds (that is hardware counters is not used).

Reply:
Sorry for the delay, i wanted to confirm with QC folks before replying. To be able to map from cycles to seconds, we need to fix the DSP vote and know the mapping from vote to DSP frequency. This is difficult as it varis by SoCs and other factors and can be modified by OEMs.

Thanks